src/main/jbake/templates/index.ftl

```diff
@@ -5,55 +5,62 @@
-            <h2>Hawkular Features</h2>
-            <div class="row">
-                <ul>
-                    <li class="col-md-4 col-sm-4">
-                        <a href="/overview/index.html#_metric_storage"><i class="fa fa-cloud"></i></a>
-                        <h4>Metric Storage</h4>
-                        <p>Flexible, scalable and high performance metric storage based on Cassandra. <a href="/overview/index.html#_metric_storage">More...</a></p>
-                    </li>
-                    <li class="col-md-4 col-sm-4">
-                        <a href="/overview/index.html#_alerting"><i class="fa fa-bell"></i></a>
-                        <h4>Alerting</h4>
-                        <p>Notify administrators of performance problems or other user defined conditions. <a href="/overview/index.html#_alerting">More...</a></p>
-                    </li>
-                    <li class="col-md-4 col-sm-4">
-                        <a href="/overview/index.html#_inventory"><i class="fa fa-sitemap"></i></a>
-                        <h4>Inventory</h4>
-                        <p>Keep track of the monitored topology and all the monitored resource metadata. <a href="/overview/index.html#_inventory">More...</a></p>
-                    </li>
-                    <li class="col-md-4 col-sm-4">
-                        <a href="/overview/index.html#_metric_data_visualization"><i class="fa fa-bar-chart"></i></a>
-                        <h4>Metric Data Visualization</h4>
-                        <p>View your data and add charts where you need them most, including real time monitoring. <a href="/overview/index.html#_metric_data_visualization">More...</a></p>
-                    </li>
-                    <li class="col-md-4 col-sm-4">
-                        <a href="/overview/index.html#_application_performance_management"><i class="fa fa-briefcase"></i></a>
-                        <h4>Application Performance Management</h4>
-                        <p>Capture information about the application invocations being executed across your servers. <a href="/hawkular-apm/index.html">More...</a></p>
-                    </li>
-                    <li class="col-md-4 col-sm-4">
-                        <a href="http://manageiq.org/"><img src="http://avatars0.githubusercontent.com/ManageIQ?&s=48" alt="ManageIQ logo"></img></a>
-                        <h4>Integration with ManageIQ</h4>
-                        <p>ManageIQ lets you manage container, virtual, private, and public cloud infrastructures. <a href="http://manageiq.org/">More...</a></p>
-                    </li>
-                </ul>
-            </div>
-        </div>
-    </section>
+</div>
+
+<div class="container">
+  <div class="row">
+    <h2>What is Hawkular?</h2>
+    <div class="paragraph">
+      <p>
+        Hawkular a hawk with a monocular. Hawks are known to have a very sharp vision and very good hunters, they can catch preys anticipating their movements at a very fast speed.
```

Comment:
What is Hawkular ?
Hawkular is a set of Open Source (Apache License v2) projects designed to be a generic solution for common monitoring problems.  The Hawkular projects provide REST services that can be used for all kinds of monitoring needs.  Our aim is to provide a generic solution that can be used for common monitoring problems.

Our project goal is to be able to monitor fast paced environments and catch anomalies quickly.  Since hawks are known to have very sharp vision and as very good hunters, anticipating their prey’s movement even at high speeds, our project logo is a hawk with a monocle.

The project started around the end of 2014 as a successor of the RHQ Project. The Hawkular project is sponsored by  Red Hat.  The monitoring services provided by Hawkular have been adopted by different projects and are central to the Middleware management solution in the ManageIQ project.